hana/test/run.test.js

```diff
@@ -0,0 +1,332 @@
+    } catch (e) {
+      if (e.message.match(/cannot use duplicate name of function or procedure/)) {
+        // ignore
+      } else {
+        throw e
+      }
+    }
+    try {
+      await cds.run(`
+        CREATE PROCEDURE ";sap. secmon~analysis#framework!pattern@getNewAndExistingAlerts" ( OUT TEST_1 NVARCHAR(32), IN VAL_2 NVARCHAR(32), OUT TEST_2 NVARCHAR(32), INOUT VAL_1 NVARCHAR(32) ) AS
+        BEGIN TEST_1 = VAL_1; TEST_2 = VAL_2; END`)
+    } catch (e) {
+      if (e.message.match(/cannot use duplicate name of function or procedure/)) {
+        // ignore
+      } else {
+        throw e
+      }
+    }
+    try {      
+      await cds.run(`
+        CREATE PROCEDURE "PROC_TEST_4" (
+          OUT TEST_1 TABLE ( ID INTEGER, "title" NVARCHAR(32) ),
+          INOUT TEST_3 INTEGER,
+          OUT TEST_2 TABLE ( ID INTEGER, "title" NVARCHAR(32) ),
+          INOUT VAL_1 INTEGER
+        ) AS BEGIN
+          TEST_1 = SELECT ID, TITLE as "title" FROM sap_capire_TestEntity WHERE ID <= :VAL_1;
+          TEST_2 = SELECT ID, TITLE as "title" FROM sap_capire_TestEntity WHERE ID > :VAL_1;
+          TEST_3 = VAL_1;
+        END
+      `)
+    } catch (e) {
+      if (e.message.match(/cannot use duplicate name of function or procedure/)) {
+        // ignore
+      } else {
+        throw e
+      }
+    }
+    try {
+      await cds.run(`
+        CREATE PROCEDURE "PROC_TEST_5" (
+          IN A INTEGER,
+          OUT B INTEGER,
+          OUT C DUMMY,
+          OUT D TABLE ( I NVARCHAR(1), J NVARCHAR(1) )
+        ) AS BEGIN
+          SELECT 'test' AS X FROM DUMMY;
+          D = SELECT 'i' AS I, 'j' AS J FROM DUMMY;
+          SELECT 2 AS Y FROM DUMMY;
+          B = :A + 1;
+          SELECT 3 AS Z FROM DUMMY;
+          C = SELECT * FROM DUMMY;
+        END
+      `)
+    } catch (e) {
+      if (e.message.match(/cannot use duplicate name of function or procedure/)) {
+        // ignore
+      } else {
+        throw e
+      }
+    }
+    try {
+      await cds.run(`
+        CREATE PROCEDURE PROC_TEST_6 ( OUT TABLE_1 TABLE ( COL_1 NVARCHAR(32) ), IN TABLE_2 TABLE ( COL_1 NVARCHAR(32) ), OUT TABLE_3 TABLE ( COL_1 NVARCHAR(32) ), INOUT VAL_1 NVARCHAR(32) ) AS
+        BEGIN TABLE_1 = SELECT VAL_1 AS COL_1 FROM DUMMY; TABLE_3 = SELECT COL_1 AS COL_1 FROM :TABLE_2; END`)
+    } catch (e) {
+      if (e.message.match(/cannot use duplicate name of function or procedure/)) {
+        // ignore
+      } else {
+        throw e
+      }
+    }
+  }
+
+  async function addData() {
+    try {   
+      await cds.run(INSERT.into('SAP_CAPIRE_TESTENTITY').columns(['ID', 'title']).rows([1, '1']))
+      await cds.run(INSERT.into('SAP_CAPIRE_TESTENTITY').columns(['ID', 'title']).rows([2, '2']))
+      await cds.run(INSERT.into('SAP_CAPIRE_TESTENTITY').columns(['ID', 'title']).rows([3, '3']))
+      await cds.run(INSERT.into('SAP_CAPIRE_TESTENTITY').columns(['ID', 'title']).rows([4, '4']))
+    } catch (e) {
+      // ignore
```

Comment:
Better notice when the data does not actually exist instead of having all tests fail with strange expectations.